dask/dataframe/__init__.py

```diff
@@ -181,10 +185,14 @@ def inner_func(*args, **kwargs):
 
 Any feedback can be reported on the Dask issue tracker
 https://github.com/dask/dask/issues 
+
+To disable this warning in the future, set dask config:
+
+    dataframe.query-planning-warning=False
```

Comment:
```suggestion
    >>> dask.config.set({'dataframe.query-planning-warning': False})
```
Changed to match the paragraph just above